spec/features/integration_spec.rb

```diff
@@ -46,7 +47,7 @@
     end
   end
 
-  it 'multiple imports successfully', vcr: { cassette_name: 'multiple_integration' } do
```

Comment:
is it failing randomly?

Reply:
no, it fails with cassette when all tests run, and works with cassette when just this test runs

Reply:
I remember that it was a problem with a single import, and suddenly now we have this problem with multiple import, I think I'll create a task to search what's going on here